build-tools/create-android-api/create-android-api.csproj

```diff
@@ -108,24 +104,24 @@
       Outputs="@(_MergedXmlFiles->'%(MergedXml)')">
     
     <PropertyGroup>
-      <ApiMerge>..\..\bin\Build$(Configuration)\api-merge.exe</ApiMerge>
+      <ApiMerge>..\..\bin\Build$(Configuration)\api-merge.dll</ApiMerge>
       <_ConfigurationFile>..\..\bin\Build$(Configuration)\merge-configuration.xml</_ConfigurationFile>
       <_ConfigurationInputBaseDirectory>..\..\bin\Build$(Configuration)\api\</_ConfigurationInputBaseDirectory>
       <_ConfigurationOutputBaseDirectory>..\..\bin\Build$(Configuration)\api\</_ConfigurationOutputBaseDirectory>
     </PropertyGroup>
     
     <Exec
-        Command="$(ManagedRuntime) $(ManagedRuntimeArgs) $(ApiMerge) -config=$(_ConfigurationFile) -config-input-dir=$(_ConfigurationInputBaseDirectory) -config-output-dir=$(_ConfigurationOutputBaseDirectory)" />
+        Command="dotnet $(ApiMerge) -config=$(_ConfigurationFile) -config-input-dir=$(_ConfigurationInputBaseDirectory) -config-output-dir=$(_ConfigurationOutputBaseDirectory)" />
```

Comment:
Should this use the `dotnet` we provision? `&quot;$(DotNetPreviewTool)&quot;`

Reply:
`TargetFramework=net7.0` wouldn't make it to be built with the system dotnet. Do we want to *run* it with the same dotnet it was *built* with?